lib/Doctrine/Inflector/Rules/Dutch/Inflectible.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Inflector\Rules\Dutch;
+
+use Doctrine\Inflector\Rules\Pattern;
+use Doctrine\Inflector\Rules\Substitution;
+use Doctrine\Inflector\Rules\Transformation;
+use Doctrine\Inflector\Rules\Word;
+
+/**
+ *  // http://nl.wikipedia.org/wiki/Klinker_(klank)
+ *  private $klinker = '(a|e|i|o|u|ij)';
+ *  private $korteKlinker = '(u|i|e|a|o)';          // @todo '(ie|oe)'. $plofKlank
+ *
+ *  // http://nl.wikipedia.org/wiki/Medeklinker
+ *  private $plofKlank = '(p|t|k|b|d)';
+ *  private $wrijfKlank = '(f|s|ch|sj|v|z|g|j)';    // journaal
+ *  private $neusKlank = '(m|n|ng)';
+ *  private $vloeiKlank = '(l|r)';
+ *  private $glijKlank = '(j|w)';                   // jaar
+ *  private $affricate = '(ts|zz|tsj|g)';           // /d3/ gin
+ *  private $missingFromWiki = '(c|h|p|q|x|y|z)';
+ *
+ *  public function __construct()
+ *  {
+ *      $this->medeklinker = '(' . $this->missingFromWiki . '|' . $this->plofKlank . '|' . $this->wrijfKlank . '|' . $this->neusKlank . '|' . $this->vloeiKlank . '|' . $this->glijKlank . '|' . $this->affricate . ')';
+ *      $this->medeklinker = '((c|h|p|q|x|y|z)|(p|t|k|b|d)|(f|s|ch|sj|v|z|g|j)|(m|n|ng)|(l|r)|(j|w)|(ts|zz|tsj|g))';
+ *  }
```

Comment:
Why is there commented out code here?

Reply:
i do not know how to get that commented code inline.
thing is the rules also count part a consonant set.

Reply:
What do you mean by inline? Why not remove all that commented code? Does it bring any value?

Reply:
may it be let in, as a remider?

Reply:
this is basic the value,

somewhere in code we original have
		'/()' . $klinker . '$/i' => '\1\2s',
		'/()(' . $medeklinker . 'e' . $medeklinker . ')$/i' => '\1\2s',
		'/()(' . $medeklinker . $korteKlinker . 's)$/i' => '\1\2sen',

see, that is a
consonant and a
vowel and a
short vowel

i want to keep the 'short vowel' concept in the code.

Reply:
It can be let in if it helps understand the rest of the code, sure 👍

Reply:
super